src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -198,17 +195,17 @@ void manualRegistrationToolBox::startManualRegistration()
     medTabbedViewContainers * tabContainers = getWorkspace()->stackedViewContainers();
     d->regOn = true;
 
-    if(this->parentToolBox()) //parentTbx only set in medRegistrationSelectorToolBox::changeCurrentToolBox
-    {
-        dtkSmartPointer<medAbstractData> fixedData(this->parentToolBox()->fixedData());
-        dtkSmartPointer<medAbstractData> movingData(this->parentToolBox()->movingData());
+    medRegistrationSelectorToolBox* toolbox = dynamic_cast<medRegistrationSelectorToolBox*>(selectorToolBox());
 
+    if(toolbox)
+    {
+        dtkSmartPointer<medAbstractData> fixedData(toolbox->fixedData());
+        dtkSmartPointer<medAbstractData> movingData(toolbox->movingData());
         if (!fixedData || !movingData)
```

Comment:
this `if` condition has no space before or after. It's a small thing but can help the reading :)